mbuild/lib/recipes/polymer.py

```diff
@@ -364,6 +384,44 @@ def add_end_groups(
             else:
                 raise ValueError("Label must be 'head' or 'tail'")
 
+    def create_periodic_bond(self, axis="z"):
+        """Align and bond the end points of a polymer along an axis.
+
+        Parameters
+        ----------
+        axis : str, default="z"
+            Axis along which to orient the polymer taken as the line connected the
+            free ports of the end group. May be "x", "y", or "z".
+        """
+        if self.head_port is None or self.tail_port is None:
+            raise ValueError(
+                "Polymer head_port and/or tail_port could not be found. Be sure to initialize"
```

Comment:
One small fix here. Add a space after `initialize"` or before "this", otherwise it shows up as `initializethis`